addons/pos_restaurant/static/src/js/floors.js

```diff
@@ -216,6 +216,15 @@ var TableWidget = PosBaseWidget.extend({
                             self.table[field] = table[field];
                         }
```

Comment:
Try replacing all the below with self.select()

Reply:
I did your suggestion with a conditional. Can you check?